composer.json

```diff
@@ -17,13 +17,14 @@
             "Boedah\\Robo\\Task\\Drush\\": "src"
         }
     },
+    "minimum-stability": "beta",
     "require": {
         "php": ">=5.5.0",
-        "consolidation/Robo": "~1"
+        "consolidation/Robo": "~1",
+        "drush/drush": "^8.0 | ^9.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.4",
-        "drush/drush": "8.1.12"
+        "phpunit/phpunit": "~4.4"
     },
     "suggest": {
         "drush/drush": "robo-drush needs a global or local Drush to use."
```

Comment:
If you move `drush/drush` from `require-dev` to `require`, then robo-drush always needs a local Drush, and will not use any global drush on the system.

If this is the desired change for this PR, then the `suggest` should be removed, as Drush is already required.

Reply:
Excellent point @greg-1-anderson , I've moved it back!